modules/abstract-utxo/src/abstractUtxoCoin.ts

```diff
@@ -1220,7 +1223,7 @@ export abstract class AbstractUtxoCoin extends BaseCoin {
     );
 
     return {
-      address,
+      address: this.canonicalAddress(address, params.format),
```

Comment:
what happens if `params.format` is `undefined` here?

Reply:
It will not do any address format conversion.